eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js

```diff
@@ -0,0 +1,88 @@
+const {
+  ERR_RIGHTSIDE_AND_OR_NOT_ALLOWED,
+  ERR_RIGHTSIDE_ALLOFF_OR_NOT_ALLOWED,
+} = require("../../../constants/validationErrorCodes");
+
+const {
+  getOptionById,
+  getExpressionGroupByExpressionId,
+} = require("../../../schema/resolvers/utils");
+const { filter } = require("lodash");
+
+module.exports = function(ajv) {
+  ajv.addKeyword("validateExclusiveCheckbox", {
+    validate: function isValid(
+      otherFields,
```

Comment:
Think you can delete this as it's not used

Reply:
I did originally, but the arguments are positional.

Reply:
ah right